docker/autorun-amc2moodle.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+## script to run amc2moodle or moodle2amc automatically in a specific folder
+
+# monitored folder
+MONITOR_DIR=/tmp/work
+
+# function to run amc2moodle
+function run_amc2moodle()
+{
+    amc2moodle -x --silent "$1"
+    touch "$1".lock
+}
+
+# function to run moodle2amc
+function run_moodle2amc()
+{
+    moodle2amc --silent "$1"
+    touch "$1".lock
+}
+
+
+#log manage
+now=$(date +"%Y-%m-%d_%H-%M-%s")
+LOGDIR=/tmp/daemon
```

Comment:
Same remark about Dockerfile ENV. At least, define `LOGDIR` on the top of the file.